commands/issue/update/issue_update.go

```diff
@@ -108,6 +159,8 @@ func NewCmdUpdate(f *cmdutils.Factory) *cobra.Command {
 	issueUpdateCmd.Flags().BoolP("public", "p", false, "Make issue public")
 	issueUpdateCmd.Flags().BoolP("confidential", "c", false, "Make issue confidential")
 	issueUpdateCmd.Flags().StringP("milestone", "m", "", "title of the milestone to assign, pass \"\" or 0 to unassign")
+	issueUpdateCmd.Flags().StringSliceP("assign", "", []string{}, "Assign new users to the issue by their `usernames`, overrides --unassign")
```

Comment:
Shouldn't we be consistent with the naming of the flags? 
`issue create` uses `--assignee` so users might expect a similar name for `issue update` without reading the docs.

Also, since the `-a` shorthand isn't taken, you should give it the `-a` shorthand.

Reply:
I have been thinking whether it makes sense to have assign and unassign to add and remove assignees.

I think it is better to have assignee (replaces current assignees).

Reply:
Definitely we can't have `--unassignee` 😂 so `unassign` makes sense.

Or maybe we can have a format like prefixing the username with `-` or any other sensible character to unassign a user:

```sh
glab issue update 12 --assignee "-profclems" --assignee "maxice8"

# unassigns @profclems and assigns @maxice8 
```

Reply:
How about `!` ? It is also used in `golang` to negate a boolean.

Reply:
and since we have `!` and `+` free, we can have.

--assignee `name` (removes all others and add `name`)
--assignee `+foo` (add `foo` on top of existing asssignees)
--assigne `-foo` (remove `foo` from existing assignees)

Reply:
`!` is also a nice one or we can support both `-` and `!` for removing assignees and the `+` for adding to existing.

Reply:
Sounds Good, I'll drop the `assign` commit from this MR and leave only the milestone and work on it separately

Reply:
I don't know if GitLab supports usernames starting with `-` (like a user has username `-anderson`) so we don't conflict with that.

Reply:
I hope they don't allow that

Reply:
I just checked. Only alphanumeric characters are supported

Reply:
me too, just logged out of `gitlab.com` to create a new account so I can test valid characters since I found no documentation on that. :laughing:

Reply:
![image](https://user-images.githubusercontent.com/41906128/103409392-7d87e880-4b5e-11eb-845d-7309a9fa9e33.png)